MusicCat/Players/AjaxAMP.cs

```diff
@@ -37,7 +40,11 @@ private Task<string> SendCommand(string command, Dictionary<string, string> args
 
         public async Task<float> GetVolume() => float.Parse(await Get("getvolume"));
 
-        public Task Pause() => Post("pause");
+#pragma warning disable CS1998 
```

Comment:
I don't want to see warnings being disabled. Async won't do anything here. Please remove `async` and wrap the function body inside a task that you start and return.